src/features/steps/SIMCard9151.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+import React from 'react';
+
+import { Back } from '../../common/Back';
+import Link from '../../common/Link';
+import Main from '../../common/Main';
+import { Next } from '../../common/Next';
+
+export default () => (
+    <Main>
+        <Main.Content heading="Activate iBasis SIM card">
+            <p>There are two pre-activated SIM cards in the kit:</p>
+            <br />
+            <ol className="tw-list-outside tw-list-decimal tw-pl-4">
+                <li>
+                    <Link
+                        label="SIM card from onomondo"
+                        href="https://onomondo.com/go/nordic-dev-kit/#form"
+                        color="tw-text-primary"
+                    />{' '}
+                    with 10MB free data for 30 days.
+                    <br />
+                    If you register the SIM card you will get additional 40MB
+                    and extend the trial period for 60 days and access the
+                    real-time network insight tool.
```

Comment:
```suggestion
                    If you register the SIM card, you will get additional 40 MB
                    and extend the trial period for 60 days and access the
                    real-time network insight tool.
```
Does this mean the trial will last 60 days total (30+30) or 90 days (30+60)?

Reply:
30+30

Reply:
```suggestion
                    If you register the SIM card, you will get additional 40 MB,
                    extend the trial period to a total of 60 days, and get access
                    to the real-time network insight tool.
```

Reply:
Edited the original suggestion.